mage_ai/data_cleaner/statistics/calculator.py

```diff
@@ -211,18 +208,22 @@ def __statistics_overview(self, series, col):
 
             if column_type not in NUMBER_TYPES:
                 if dates is not None:
-                    value_counts = dates.value_counts()
-                else:
-                    value_counts = series_non_null.value_counts()
-
-                mode = value_counts.index[0]
-                if column_type == DATETIME:
+                    df_value_counts = dates.value_counts()
+                    data[f'{col}/value_counts'] = (
```

Comment:
For datetime column, we need to convert the index to str. Otherwise, there'll be json serialization issue when writing stats to disk.
https://github.com/mage-ai/mage-ai/blame/master/mage_ai/data_cleaner/statistics/calculator.py#L158

Reply:
https://github.com/mage-ai/mage-ai/pull/220

Reply:
I accidentally tried converting the value counts instead of the index, fixed in this new PR.